packages/validators/package.json

```
@@ -0,0 +1,40 @@
+{
+  "name": "@contentful/experiences-validators",
+  "private": true,
+  "version": "0.0.0",
+  "main": "./dist/index.js",
+  "module": "./dist/index.js",
+  "types": "./dist/types.d.ts",
```

```suggestion
  "types": "./dist/index.d.ts",
```
See in the other comments on `typesVersions` and `exports` why this might make more sense.